.github/workflows/_reusable_run_tests.yml

```diff
@@ -124,14 +124,17 @@ jobs:
           bundle exec fastlane prepare_for_tests
 
       - name: Install 1Password
+        if: ${{ inputs.test_dependencies == false && github.base_ref != 'release/cycle-3.120' && github.base_ref != 'release/cycle-3.124' }}
+
         uses: 1password/install-cli-action@v1
 
       - name: Generate env file
+        if: ${{ inputs.test_dependencies == false && github.base_ref != 'release/cycle-3.120' && github.base_ref != 'release/cycle-3.124' }}
         run: |
           bundle exec fastlane generate_env
 
       - name: "Test new critical flows"
-        if: ${{ inputs.test_dependencies == false }}
+        if: ${{ inputs.test_dependencies == false && github.base_ref != 'release/cycle-3.120' && github.base_ref != 'release/cycle-3.124' }}
```

Comment:
suggestion: leave a note in `.github/workflows/test_develop.yml` so that we clean up `3.124` here when 3.124 is removed from the nightly test runs.

Reply:
fixed